logdna/index.js

```diff
@@ -106,13 +99,15 @@ const sendLine = (payload, config, callback) => {
     // Flush the Log
     asyncRetry({
         times: MAX_REQUEST_RETRIES
-        , interval: REQUEST_RETRY_INTERVAL_MS
+        , interval: (retryCount) => {
+          return REQUEST_RETRY_INTERVAL_MS * Math.pow(2, retryCount);
+        }
         , errorFilter: (err) => {
-            return err.code === 'ETIMEDOUT' || err.code === 'ESOCKETTIMEDOUT';
+            return DEFAULT_HTTP_ERRORS.includes(err.code);
         }
     }, (reqCallback) => {
         return request(options, (error, response, body) => {
-            if (error) return reqCallback(error);
+            if (error || response.statusCode >== INTERNAL_SERVER_ERROR) return reqCallback(error);
```

Comment:
Does it retry if `response.statusCode >== INTERNAL_SERVER_ERROR`? I don't think so.